contracts/utils/cryptography/P256.sol

```diff
@@ -299,10 +299,10 @@ library P256 {
         points[0x09] = _jAddPoint(points[0x01], points[0x08]); // 1,2 (p+2g)
         points[0x0a] = _jAddPoint(points[0x02], points[0x08]); // 2,2 (2p+2g)
         points[0x0b] = _jAddPoint(points[0x03], points[0x08]); // 3,2 (3p+2g)
-        points[0x0c] = _jAddPoint(points[0x04], points[0x08]); // 0,3 (g+2g)
+        points[0x0c] = _jAddPoint(points[0x04], points[0x08]); // 0,3 (3g)
```

Comment:
This comment attempted to signal that the result of the `0,3` cell is that of `points[0x04]` (g) + `points[0x00]` (2g) so I'm afraid that by rewriting it to `3g` it loses its purpose. No strong opinion but I'd rather do:

```suggestion
        points[0x0c] = _jAddPoint(points[0x04], points[0x08]); // 0,3 (g+2g = 3g)
```

Reply:
Note: whatever we decide, this line should be consistent with line 295

Reply:
updated both in lines https://github.com/OpenZeppelin/openzeppelin-contracts/pull/5206/commits/6eb8c245fea5de371a01edc881d55f3c626adc8e